app/controllers/histories_controller.rb

```diff
@@ -8,9 +8,45 @@ class HistoriesController < ApplicationController
   def create
     redirect_to root_url unless current_user.can_create_subject_history?
 
-    History.create!(patient_id: params.permit(:patient_id)[:patient_id],
+    history = History.new(patient_id: params.permit(:patient_id)[:patient_id],
+                          created_by: current_user.email,
+                          comment: params.permit(:comment)[:comment],
+                          history_type: params.permit(:type)[:type] || 'Comment')
+
+    history.original_comment = history
```

Comment:
should we only populate this field for `comments` or are we thinking of having revisions for other history types in the future too?

Reply:
spoke with @holmesie and we want to leave this blank for non-comment histories